client/my-sites/plugins/plugins-list/index.jsx

```diff
@@ -280,15 +301,25 @@ export class PluginsList extends Component {
 					// only consider the sites where the those plugins are installed
 					.filter( ( [ , sitePlugin ] ) => sitePlugin.update?.new_version )
 					.forEach( ( [ siteId ] ) => {
+						updatedPlugins.add( plugin.slug );
+						updatedSites.add( siteId );
+
 						const sitePlugin = this.getSitePlugin( plugin, siteId );
 						return this.props.updatePlugin( siteId, sitePlugin );
 					} );
 			} );
+
+		recordTracksEvent( 'calypso_plugins_bulk_action_execute', {
+			action: 'updating',
+			plugins: [ ...updatedPlugins ].join( ',' ),
+			sites: [ ...updatedSites ].join( ',' ),
+		} );
```

Comment:
When I visit: http://jetpack.cloud.localhost:3001/plugins/manage

And I click the inline "Update" button:

![image](https://user-images.githubusercontent.com/1749918/190629604-ca7d6d35-9e5d-4937-bf18-7d2d306f5cdc.png)

Is it expected to see the following two events recorded:
- calypso_plugins_bulk_action_execute
- calypso_plugin_update_click

Reply:
I do see both of these events get recorded with no errors. Under what circumstances are you not seeing these events?

Reply:
Yes, I also see both events recorded, and I'm curious if that's intended. If you click the inline update button, should we also count it as a bulk action?

Reply:
Oh I understand now. The calypso_plugin_update_click was left over from an iteration where I was having the events be on each individual button. I have removed it.